iotedgedev/cli.py

```diff
@@ -262,19 +262,24 @@ def deploy(manifest_file):
 @click.option("--name",
               "-n",
               required=True,
-              help="Specify the deployment name")
+              help=("Specify the name of the deployment that will be created in the IoT hub. "
+                    "Give your deployment a unique name that is up to 128 lowercase letters. Avoid spaces and the following invalid characters: & ^ [ ] { } \\ | \" < > /."))
```

Comment:
>that is up to 128 lowercase letters. Avoid spaces and the following invalid characters: & ^ [ ] { } \\ | \" < > /." [](http://example.com/codeflow?start=56&length=115)

should we instead point to the msdn doc that lists these limitations in case they change in the future? iotedgedev isn't doing the validation explciitely so I thought we should delegate the definition too :)

Reply:
Right, good point, I'll replace with link